src/components/Sidebar/components/SubNavList/utils.js

```diff
@@ -0,0 +1,6 @@
+import { isArray } from '../../../../util/type-check';
+
+const getSelectedChildIndex = children =>
+  isArray(children) ? children.findIndex(child => child.props.selected) : 0;
```

Comment:
`findIndex` is not supported in IE. Luckily, `lodash` provides `findIndex` as well: https://lodash.com/docs/4.17.11#findIndex

Reply:
Fixed. Thanks a lot!